pages/common/just.md

```diff
@@ -0,0 +1,24 @@
+# just
+
+> A handy way to save and run project-specific commands.
+> More information: <https://github.com/casey/just>.
+
+- Run a recipe specified in the justfile:
+
+`just {{recipe}}`
+
+- Initialize new justfile in project root:
+
+`just --init`
+
+- Edit justfile with editor given by $VISUAL or $EDITOR, falling back to vim:
```

Comment:
We used to use this phrase
```suggestion
- Edit justfile in the default editor:
```
Is the `justfile` is the current directory?

Reply:
just is smart enough to find justfile in the current project root (I think it uses git)

Reply:
otherwise current directory, yes